fiat-constify/src/main.rs

```diff
@@ -59,29 +59,33 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
 
 /// Build a toplevel attribute with the given name and comma-separated values.
 fn build_attribute(name: &str, values: &[&str]) -> Attribute {
-    let span = Span::call_site();
     let values = values
         .iter()
         .map(|value| build_path(value))
         .collect::<Vec<_>>();
+    let path = build_path(name);
+    let tokens = quote! { (#(#values),*) };
```

Comment:
I confirmed locally this works:

```suggestion
    let tokens = quote! { #(#values),* };
```

Reply:
You're right, the data structure changed between the two versions and I missed it, thanks.